packages/api/src/command/medical/patient/create-patient.ts

```diff
@@ -42,5 +44,12 @@ export const createPatient = async (patient: PatientCreateCmd): Promise<Patient>
   // Intentionally asynchronous - it takes too long to perform
   cwCommands.patient.create(newPatient, facilityId).catch(processAsyncError(`cw.patient.create`));
 
+  // TODO: NEED CQ PATIENT DIRECTORY SEARCH
+  // const organization = await getOrganizationOrFail({ cxId, id: facilityId });
```

Comment:
The facility your right is unnecessary as for the location i figured since this was logic specific to the domain we would use it here and not on the cq level